Libplanet/Tx/Transaction.cs

```diff
@@ -460,31 +466,40 @@ public byte[] ToBencodex(bool sign)
         /// <c>false</c>.
         /// </exception>
         [Pure]
-        public IAccountStateDelta EvaluateActions(
+        public IEnumerable<(T, IActionContext, IAccountStateDelta)>
+        EvaluateActionsGradually(
             HashDigest<SHA256> blockHash,
             long blockIndex,
             IAccountStateDelta previousStates,
             Address minerAddress,
             bool rehearsal = false
         )
         {
+            ActionContext CreateActionContext(
+                IAccountStateDelta prevStates,
+                int randomSeed
+            ) =>
+                new ActionContext(
+                    signer: Signer,
+                    miner: minerAddress,
+                    blockIndex: blockIndex,
+                    previousStates: prevStates,
+                    randomSeed: randomSeed,
```

Comment:
Is it okay not to have `unchecked`?

Reply:
Fixed: 604f9b57060498eecd3c03783be742c39f0260af.